sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -2815,7 +2821,7 @@ export class ServiceBusAdministrationClient extends ServiceClient {
     try {
       const rule = buildRule(response.parsedBody);
       const ruleResponse: WithResponse<RuleProperties> = Object.assign(rule || {}, {
-        _response: response
+        _response: getHttpResponseOnly(response)
```

Comment:
When you assign it this way does it make _response non-enumerable? I believe generated clients do this today.

Reply:
Thanks, It doesn't.
Should be doing `Object.defineProperty(obj, "_res", { value : "res" });` instead.

[Playground Link](https://www.typescriptlang.org/play?#code/MYewdgzgLgBCBGArAvAbwIYC4CMAaemATAL4DcAsAFADySApsFAHToQQCWA5mABQKIwAPoJipiuUQH0ATnQiYARLIgLiASgqVQkEABs6TXSE58kGqlW3Q4SQmix4CJTbUQNmrDt1OJCQkWISqDJyisqq5lrgEHoGRiaoTEn8JJFUAPTpMAAqABbsEDAQuSAArroAJjDwdDDSXLlQltGw-ADM9jj4RGRUru5MFXQAZuxgdAAK0iAADnTSUACePm0SCiEquKgAbui6pXRhchGaVrGGxjyJyUht6qRAA)

Reply:
Made it